web-module/src/main/webapp/resources/structEditJS/dateTime/celements-date-time-field.js

```diff
@@ -0,0 +1,461 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+(function(window, undefined) {
+  'use strict';
+  
+  const versionTimeStamp = new Date().getTime();
+  const curScriptElement = document.currentScript;
+  const curScriptPath = curScriptElement.src.split('?')[0];
+  const curScriptDir = curScriptPath.split('/').slice(0, -1).join('/')+'/';
+
+  if(typeof window.CELEMENTS.structEdit=="undefined"){window.CELEMENTS.structEdit={};};
+
+  if (typeof window.CELEMENTS.structEdit.DateOrTimeFieldPicker === 'undefined') {
+    window.CELEMENTS.structEdit.DateOrTimeFieldPicker = Class.create({
+      FIELD_CHANGED : 'celements:fieldChanged',
+      _inputField : undefined,
+      _buttonCssSelector : undefined,
+      _defaultFormat : undefined,
+      _pickerConfigObj : undefined,
+      _fieldValidator : undefined,
+      _onShowBind : undefined,
+      _onChangedBind : undefined,
+      _onChangeFieldBind : undefined,
+      _openPickerNow : undefined,
+      _pickerButtonClickHandlerBind : undefined,
+      _pickerButton : undefined,
+
+      initialize : function(inputField, buttonCssSelector,
+          defaultFormat, pickerConfigObj, fieldValidator){
+        var _me = this;

```

Comment:
check `const`/`let` usage in the whole file.
```suggestion
        const _me = this;
```